README.md

```diff
@@ -24,13 +28,18 @@ Please add your project below if it uses this infrastructure for translations:
 
 ## Installation
 
-Run
+- ### [mdbook-i18n-helpers](./i18n-helpers/README.md):
```

Comment:
Please avoid headings inside list items.

Reply:
Basically, I don't think the text below should be a huge list item, it should just be a normal un-indented `###` section.